db/migrate/20160816144624_create_damages.rb

```diff
@@ -0,0 +1,14 @@
+class CreateDamages < ActiveRecord::Migration
+  def change
+    create_table :damages do |t|
+      t.string :uuid
+      t.string :damage_issued_reservation_uuid
```

Comment:
I'm not sure I understand the question - what I mean here is that if we just call the column `damage_issued_reservation`, it will be understood that the format of that data is a UUID, since it refers to a `Reservation` object, which exists in aggressive-inventory, not this app.